src/main/java/uk/nhs/hee/tis/revalidation/connection/dto/ConnectionDto.java

```diff
@@ -0,0 +1,19 @@
+package uk.nhs.hee.tis.revalidation.connection.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import io.swagger.annotations.ApiModel;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+@ApiModel(description = "List of connection information")
+public class ConnectionDto {
+  private List<ConnectionHistoryDto> connectionHistoryDtos;
+}
```

Comment:
This is interesting.  Is there a particular use case when referencing a `List<ConnectionHistoryDto>` doesn't work as well?

Reply:
It was also working. But was discussed with the team, and changed it to keep the practice for the frontend to consume and keep it flexible if we need to add count/total results in the future.